### ISSUE TITLE
field account-blocklist not found

### ISSUE THREAD

**Issue by den-is** [rebuy-de/aws-nuke#600]
Weird YAML unmarshal issue I'm getting while trying to run aws-nuke for the first time.
aws-nuke v2.15.0-rc.2 and v2.14.0
macos catalina 10.15.7

```yaml
regions:
- ca-central-1
- global

account-blocklist:
- "054750000000" # prod

accounts:
  "675181111111111": {} # sandbox
```
_accounts values are intentionally altered_

```
aws-nuke -c config/sandbox.yaml
ERRO[0000] Failed to parse config file config/sandbox.yaml 
Error: yaml: unmarshal errors:
  line 5: field account-blocklist not found in type config.Nuke
```

Please help. have no idea what is going on

**Comment by bjoernhaeuser** [comment]
@den-is You need to check the documentation for the version you are using: https://github.com/rebuy-de/aws-nuke/tree/v2.14.0#usage

2.14.0 is still using the term "account-bl_a_cklist", the name change is not yet released. Hope this helps.

**Comment by der-eismann** [comment]
I just released v2.15.0-rc.3 where you can use the blocklist.

**Comment by orihomie** [comment]
Hi
Thanks for the tool and please, update the docker part of readme as it uses bugged v2.11
